go.mod

```diff
@@ -4,7 +4,7 @@ go 1.13
 
 replace (
 	github.com/docker/distribution v2.7.1+incompatible => github.com/docker/distribution v2.7.1-0.20190205005809-0d3efadf0154+incompatible
-	github.com/firecracker-microvm/firecracker-go-sdk v0.19.0 => github.com/smira/firecracker-go-sdk v0.19.1-0.20200110185541-4fce8cba9f84
+	github.com/firecracker-microvm/firecracker-go-sdk v0.19.0 => github.com/smira/firecracker-go-sdk v0.19.1-0.20200124123725-fb8b25794e34
```

Comment:
Maybe I am misunderstanding but the PR comment says this is supposed to move to the official SDK?

Reply:
yeah, it's still a branch in my fork with three PRs mentioned above merged together, previous branch contained some hacks I did to pass `ip=` manually and validation was disabled, so this is much cleaner version, if/when those PRs are merged, we can just switch to master now

Reply:
Ahh gotcha.